src/selector.rs

```diff
@@ -28,9 +28,10 @@ pub trait SelectedBy<T> {
 /// ```
 /// use foxbox_taxonomy::selector::*;
 /// use foxbox_taxonomy::services::*;
+/// use foxbox_taxonomy::util::Id;
 ///
 /// let selector = ServiceSelector::new()
-///   .with_tags(vec!["entrance".to_owned()])
+///   .with_tags(vec![Id::<TagId>::new("entrance".to_owned())])
```

Comment:
No, the compiler complains : "error: unable to infer enough type information about `_`; type annotations or generic parameter binding required [E0282]"